content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1087,6 +1087,22 @@ Used on: IngressClass
 When a IngressClass resource has this annotation set to `"true"`, new Ingress resource
 without a class specified will be assigned this default class.
 
+### nginx.ingress.kubernetes.io/configuration-snippet
+
+Type: Annotation
+
+Example: `nginx.ingress.kubernetes.io/configuration-snippet: |
+  more_set_headers "Request-Id: $req_id";`
+
+Used on: Ingress
+
+With use of this annotation someone can add additional configuration to the
+Nginx location as mentioned in example above. `configuration-snippet` annotation is disabled
```

Comment:
I'd signpost readers to read the ingress controller's docs for the full details and an example.